src/mcts/search.cc

```diff
@@ -1162,13 +1163,11 @@ SearchWorker::NodeToProcess SearchWorker::PickNodeToExtend(
         const auto d = node->GetD();
         const auto m = node->GetM();
         const auto terminal_visits = node->GetN();
-        LOGFILE << "== reverting " << terminal_visits << " visits to " <<
-              "a twofold terminal at depth " << depth - 1 << " ==";
         for (Node* node_to_revert = node; node_to_revert != nullptr;
                         node_to_revert = node_to_revert->GetParent()) {
           // Revert all visits on twofold draw when making it non terminal.
-          node_to_revert->RevertTerminalVisits(wl, d,
-                          m + (float)depth_counter, terminal_visits);
+          node_to_revert->RevertTerminalVisits(wl, d, m + (float)depth_counter,
+                                                terminal_visits);
```

Comment:
This still looks wrong - I also think the for statement wrapping is non-standard too.

Reply:
Done.